kotti/resources.py

```diff
@@ -801,7 +844,13 @@ def default_get_root(request=None):
             this will be an instance of :class:`~kotti.resources.Document`.
     """
 
-    return DBSession.query(Node).filter(Node.parent_id == None).one()  # noqa
+    # Get the root cache, make one if it doesn't exist yet.
+    try:
+        drc = get_settings()['kotti.default_root_cache']
+    except KeyError:
+        drc = get_settings()['kotti.default_root_cache'] = DefaultRootCache()
```

Comment:
I don't think we need another setting here as `default_get_root` if overridable already.